q2_quality_control/quality_control.py

```diff
@@ -32,21 +29,26 @@ def exclude_seqs(feature_sequences: DNAFASTAFormat,
     query_series = _dnafastaformats_to_series(feature_sequences)
 
     # if no hits are in res, early return empty hits and query_series as misses
```

Comment:
Your comment still references the old variable name, `res`.

Reply:
thanks for catching!